tests/mashupDocumentParser.test.ts

```diff
@@ -0,0 +1,36 @@
+import MashupHandler from "../src/mashupDocumentParser";
+import { arrayUtils, pqUtils } from "../src/utils";
+import { simpleQuery, section1mMock } from "./mocks";
+import base64 from "base64-js";
+import JSZip from "jszip";
+import WorkbookTemplate from "../src/workbookTemplate";
+import { section1mPath } from "../src/constants";
+describe("Mashup Document Parser tests", () => {
+    test("ReplaceSingleQuery test", async () => {
+        const mashupHandler = new MashupHandler();
+
+        const defaultZipFile = await JSZip.loadAsync(
+            WorkbookTemplate.SIMPLE_QUERY_WORKBOOK_TEMPLATE,
+            { base64: true }
+        );
+        const base64str = await pqUtils.getBase64(defaultZipFile);
+
+        const replacedQueryStr = await mashupHandler.ReplaceSingleQuery(
```

Comment:
nit - maybe have the name indicate this is a base64 string (might be confused with the query string itself)